components/ai/code/ai-bbe/natural_programming_vscode_features.md

```diff
@@ -0,0 +1,12 @@
+---
+title: "Natural Programming: Bridge Requirements and Code"
+description: "Powered by integrating Generative AI into the programming system, natural programming bridges requirements, documentation, and code to enable continued alignment of the implementation with the requirements.
+
+- **AI-Powered Code Generation** – Generate code against high-level business user requirements.
+
+- **AI-Powered Test Generation** – Generate Ballerina tests for the implementation against business user requirements, enabling identification of implementation deviation from the requirements.
+
+- **Drift Detection & Fixes** – Proactively identify drift between requirements, code, and documentation. Use smart fix suggestions via VS Code to review and accept AI-powered suggestions to address drift.  
+
+- **Intent Preservation** – Maintain an effective summary of the prompts used to generate code, providing insight into the developers' input."
```

Comment:
Looks better with new lines between the points IMO

<img width="1344" alt="image" src="https://github.com/user-attachments/assets/7257956c-4995-49aa-9d54-4137195412da" />